config.toml.example

```diff
@@ -171,37 +176,37 @@ changelog-seen = 2
 # first compiler.
 #
 # Defaults to host platform
-#build = "x86_64-unknown-linux-gnu"
+#build = <host platform>
 
 # Which triples to produce a compiler toolchain for. Each of these triples will
 # be bootstrapped from the build triple themselves.
 #
 # Defaults to just the build triple
-#host = ["x86_64-unknown-linux-gnu"]
+#host = [build]
 
 # Which triples to build libraries (core/alloc/std/test/proc_macro) for. Each of
 # these triples will be bootstrapped from the build triple themselves.
 #
 # Defaults to `host`. If you set this explicitly, you likely want to add all
 # host triples to this list as well in order for those host toolchains to be
 # able to compile programs for their native target.
-#target = ["x86_64-unknown-linux-gnu"]
+#target = host
 
 # Use this directory to store build artifacts.
 # You can use "$ROOT" to indicate the root of the git repository.
 #build-dir = "build"
 
 # Instead of downloading the src/stage0.txt version of Cargo specified, use
 # this Cargo binary instead to build all Rust code
-#cargo = "/path/to/bin/cargo"
+#cargo = build-dir/<host platform>/"stage0/bin/cargo"
```

Comment:
The location of the " is a bit weird.

Reply:
Hmm, I agree but I'm also not sure how to make it more clear. `build-dir` and `<host platform>` are both variables, while the rest is a literal string.

Reply:
Maybe
```suggestion
#cargo = "<build-dir>/<host platform>/stage0/bin/cargo"
```

Reply:
Sounds good, thanks :)